ray_data_eval/image_training/image_loader_microbenchmark.py

```diff
@@ -546,10 +557,15 @@ def collate_fn(examples):
             )
 
         # ray.data, with transform.
-        ray_dataset = ray.data.read_images(args.data_root, mode="RGB").map(crop_and_flip_image)
+        ray_dataset = ray.data.read_images(
+            args.data_root, mode="RGB", transform=get_transform(False)
```

Comment:
Add a note on the `transform=` argument that this only works with our custom fork of Ray